site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -69,7 +69,7 @@
         "id": "hRTa3Ee15WsJ"
```

Comment:
<blockquote>Learn more about image preprocessing.</blockquote><p> </p><p>Just like this, it comes across as a command, instead of as an option. "you can learn more there" vs. "learn more!".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/2081/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Thanks. Done.</p>